packages/react-cosmos/src/config/createCosmosConfig.ts

```diff
@@ -12,6 +12,8 @@ export function createCosmosConfig(
     rootDir,
     exportPath: getExportPath(cosmosConfigInput, rootDir),
     staticPath: getStaticPath(cosmosConfigInput, rootDir),
+    httpsKeyPath: getHttpsKeyPath(cosmosConfigInput, rootDir),
+    httpsCertPath: getHttpsCertPath(cosmosConfigInput, rootDir),
```

Comment:
I would prefer this mirror the `CosmosConfigInput` shape to avoid confusion when working with the config. By this I mean nesting the `httpsOptions` here as well, especially since you can either have none or both, having just one doesn't make sense.